newIDE/app/src/CommandPalette/CommandPalette/AutocompletePicker.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import * as React from 'react';
+import { type I18n } from '@lingui/core';
```

Comment:
Use `import { type I18n as I18nType } from '@lingui/core';` for consistency with the rest of the codebase